momento-sdk/src/main/java/momento/sdk/messages/CacheListFetchResponse.java

```diff
@@ -0,0 +1,106 @@
+package momento.sdk.messages;
+
+import com.google.protobuf.ByteString;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+import momento.sdk.exceptions.SdkException;
+import momento.sdk.internal.StringHelpers;
+
+/**
+ * Response for a cache list fetch operation indicating requested data was available in the cache
+ */
+public interface CacheListFetchResponse {
+
+  /** A successful cache list fetch operation. */
+  class Hit implements CacheListFetchResponse {
+    private final List<ByteString> values;
+
+    /**
+     * Constructs a cache list fetch hit with an encoded value.
+     *
+     * @param values the retrieved value.
+     */
+    public Hit(List<ByteString> values) {
+      this.values = values;
+    }
+
+    /**
+     * Gets the retrieved values as a byte array.
+     *
+     * @return the values.
+     */
+    public List<byte[]> valueByteArrayList() {
+      List<byte[]> valuesAsByteArray = new ArrayList<>();
+      for (int i = 0; i < values.size(); i++) {
```

Comment:
Any reason you do it this way vs a stream?
`values.stream().map(ByteString::toByteArray).collect(Collectors.toList()))`
I think a stream can be slower in some cases.

There is also a question of whether we should perform this computation at constructor time vs doing it here vs memoizing it. @gautamomento @malandis any opinions? Guava has a Suppliers.memoize function and we are stuck with a guava dependency.